app/components/affiliate-unit.js

```diff
@@ -23,6 +23,17 @@ export default Component.extend(
       return this.i18n.t('affiliate-unit.big-unit-heading');
     }),
 
+    getUnitLink: computed('link', function () {
+      let unitLink = this.unit.link;
+
+      if (this.isInContent && this.unit.links.page) {
```

Comment:
Make sure you check if `unit.links` exists (same below)

Reply:
Also, had to throw in a check for `!this.isInContent`.